README.md

```diff
@@ -287,11 +287,30 @@ Normal build is
 
     cmake .. -DCMAKE_TOOLCHAIN_FILE=~/mjbots/kodlab_mjbots_sdk/cmake/pi.cmake
 
+
+
 # Running Code
 To Run code, first scp the binary onto the pi, and then Run it as sudo using:
 
     sudo ./code_binary
 
+# Troubleshooting 
```

Comment:
Probably this should not be at this level since its troubleshooting the setup.

Reply:
Should I shift it one section above to, right after the build?

Reply:
The number of `#` denotes the level, so move it to `##`, then put it under in the lines after `motor setup` which is the last thing beneath `setup`

Reply:
Okay will do that